utils/build.gradle

```diff
@@ -0,0 +1,8 @@
+plugins {
+    id 'printscript.gradle.kotlin-common-conventions'
+}
+
+dependencies {
+    implementation project(':common')
+    implementation project(':lexer')
+}
```

Comment:
I don't see the point of having it in an utils module. Will it be used for other modules too?

Reply:
It didn't make sense to me to add it in the lexer module, since it is more of a utility class rather than a lexer. If you think it should go in the lexer module or somewhere else I can change it